docs/naming_conventions.md

```diff
@@ -150,3 +150,37 @@ error=UpperCamelCase
 error.suffix=Error
 
 ```
+
+Default Dart names
+----------
+
+### General naming conventions
+* Type names are in UpperCamelCase.
+* Method and variable names are in lowerCamelCase.
+* Constant and enumerator names are in lowerCamelCase.
+
+### Property names
+Properties are generated as properties in Dart:
+* Property names have no prefix, unless it's a Boolean property.
+* Boolean property names are prefixed with `is`.
```

Comment:
> Boolean property names are prefixed with `is`.

I would write: Boolean property names may have prefix `is` or `has/have`

Reply:
No, automatic naming only adds "is", there's no logic for "has/have".